client/src/app/routes/stardust/NFTDetails.tsx

```diff
@@ -293,6 +307,27 @@ class NFTDetails extends AsyncComponent<RouteComponentProps<NFTDetailsRouteProps
         return this.activityHistory.slice(firstPageIndex, lastPageIndex);
     }
 
+    private async getNftDetails() {
+        const networkId = this.props.match.params.network;
+        const nftId = this.props.match.params.nftId;
+
+        const nftOutputs = await this._tangleCacheService.nftDetails({
+            network: networkId,
+            nftId
+        });
+
+        if (nftOutputs?.outputs && nftOutputs?.outputs?.items.length > 0) {
+            for (const outputId of nftOutputs.outputs.items) {
+                const output = await this._tangleCacheService.outputDetails(networkId, outputId);
+
+                if (output && !output.isSpent && output.output.type === NFT_OUTPUT_TYPE) {
+                    const nftDetails = output.output;
+                    // TODO: Code here to map the data on the page
```

Comment:
Missing something here ?

Reply:
Map and displayed amount and nftId which we get in nftDetails object.

Reply:
Map and displayed amount and nftId which we get in nftDetails object.